tests/conftest.py

```diff
@@ -84,6 +84,7 @@ def print_test_values(
     11.0,
 ]
 N_WIND_SPEEDS = len(WIND_SPEEDS)
+N_FINDEX = N_WIND_SPEEDS
```

Comment:
We should decide how we want to demonstrate the 0th column. To match the previous tests, this would be some combination of the wind speeds and wind directions.

Reply:
You mean in the examples?

Reply:
You covered it by adding a comment, thanks